LICENSE-3RD-PARTY.txt

```diff
@@ -405,135 +405,6 @@ A library to handle automated deprecations
    limitations under the License.
 
 
-howso-openapi-client
```

Comment:
Why was this removed? Isn't a depends?

Reply:
Yes, but not 3rd party 🙂 we don't typically put Howso software in this file (or include it in the xray scan)

Reply:
Ooo, yeah, makes sense!